data/migrations/20211102125418_courses.js

```diff
@@ -10,6 +10,9 @@ exports.up = (knex) => {
     .createTable('classes', (table) => {
       table.increments('class_id');
       table.integer('size').notNullable();
+      table.integer('min_age');
+      table.integer('max_age');
+      table.integer('sessions');
       table.integer('open_seats_remaining');
```

Comment:
could you please add a day field (days of the week), and a course_type_id that references the course_types table